libs/shared/src/components/UX/Input/index.tsx

```diff
@@ -46,7 +46,7 @@ export const UxInput = (props: BaseProps) => {
     else {
       localChange(value);
     }
-  }, [forceValidate, onValidate])
```

Comment:
because onValidate could be a local fn (in some forms) it could change every re-render

Reply:
if forceValidate was `true`, then the re-render would trigger re-validation, which would then cause a re-render etc.